packages/demo-html/public/widget-with-params-js.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Static HTML Demo</title>
+    <style>
+      #wrapper {
+        width: 600px;
+        height: 400px;
+        margin: 0 auto;
+      }
+    </style>
+    <link rel="stylesheet" href="./lib/css/widget.css" />
+  </head>
+  <body>
+    <div id="wrapper"></div>
+    <script src="./lib/embed-next.js"></script>
+    <script>
+      window.tf.createWidget("moe6aa", {
+        container: document.getElementById("wrapper"),
+        source: "localhost",
+        transferableUrlParameters: ["foo"],
```

Comment:
Can we do an example of using pure HTML (in `demo-html.html`) too?

Since this is an array you would need to extend `Transformation` in `load-options-from-attributes.ts` to also include `'array'` and parse an array out of a coma separated list - add new `case` to `transformAttributeValue`. Then the demo would look like this:

```
<div ... data-tf-transferable-url-parameters="foo,bar"></div>
```

Reply:
cool!